packages/suite/src/views/wallet/details/index.tsx

```diff
@@ -74,12 +76,16 @@ const Details = () => {
     const accountTypeUrl = getAccountTypeUrl(account.path);
     const accountTypeDesc = getAccountTypeDesc(account.path);
 
+    const isCoinjoinAccount = account.backendType === 'coinjoin';
```

Comment:
I don't understand why type-check is failing here https://github.com/trezor/trezor-suite/actions/runs/3631950201/jobs/6139355172#step:14:230 🤷